### PR TITLE
[Documentation] Alcor performance test plan

### PR DESCRIPTION
**Pull request by xieus**
This PR propose Alcor performance test plan for v0.17 release. It includes the following sessions:

- Overview
- Test Environment
  - Overall Setup
  - Hardware
  - OS/Software Version
  - Benchmarking Tool
- Performance Metrics and Test Strategy
- Test Cases
  - Test Suite 1: Basic Test Suite
  - Test Suite 2: Stressful Test Suite
  - Test Suite 3: Scalability Test with Many Networks
  - Test Suite 4: Scalability Test with Large Network
  - Test Suite 5: Scalability Test with Mixed Network Size
- System Tracing
  - A Example of Alcor Trace
  - Target User Scenario
- System Monitoring
- Reports
- References

### CODE REVIEW

**Comment by xieus** [comment]
@cj-chung Can you take a look at this PR? Thanks.

File: docs/test/perfTestPlan.adoc [hunk 1]
```diff
@@ -0,0 +1,327 @@
...
+== Test Environment
+
+This section includes preparation and details of our test environment including hardware specification of servers,
+network parameters, operation system and perf test tools.
+
+=== Overall Setup
+
+Our performance test environment contains 3 types of nodes:
+
+- Controller nodes: hosting both OpenStack control plane, Alcor control plane and ElasticSearch analytics cluster
```
Comment (cj-chung) [inline docs/test/perfTestPlan.adoc:36]: kubernetes cluster for Alcor control plane, ElasticSearch analytics and Jaeger tracing cluster

**Review by cj-chung — APPROVED** [review]
one minor comment, others LGTM

File: docs/test/perfTestPlan.adoc (hunk 1 above)
Comment (xieus) [reply docs/test/perfTestPlan.adoc]: Fixed.

**Pull request merged**